server/src/main/java/org/elasticsearch/action/support/master/info/ClusterInfoRequest.java

```diff
@@ -27,11 +28,22 @@ public abstract class ClusterInfoRequest<Request extends ClusterInfoRequest<Requ
 
     private IndicesOptions indicesOptions = IndicesOptions.strictExpandOpen();
 
+    public ClusterInfoRequest(TimeValue masterTimeout) {
+        super(masterTimeout);
+    }
+
+    public ClusterInfoRequest(TimeValue masterTimeout, IndicesOptions indicesOptions) {
+        super(masterTimeout);
+        this.indicesOptions = indicesOptions;
+    }
+
+    @Deprecated(forRemoval = true)
     public ClusterInfoRequest() {
         super(TRAPPY_IMPLICIT_DEFAULT_MASTER_NODE_TIMEOUT);
     }
 
     // So subclasses can override the default indices options, if needed
+    @Deprecated(forRemoval = true)
     protected ClusterInfoRequest(IndicesOptions indicesOptions) {
```

Comment:
I think there's only a couple more uses of these, right? And then we can remove them?

Reply:
Yep, the three PRs I opened yesterday remove all usages of these deprecated constructors. I added the annotation to make it easier to see deprecated usages, but realized later I didn't actually need to commit them. They'll be automatically removed when I remove the constructors.